README.md

```diff
@@ -43,59 +44,94 @@ Contains the 3D models of the external components and their supports
-  - **diode_schottky.SLDPRT**: Schottky diode.
+  - **bornier.SLDPRT**: Screw terminals block.
   - **REG_313080006.SLDPRT**: 313080006 5V regulator
-  - **relay.SLDPRT**: Relay
+  - **Relays**
+    - **relay.SLDPRT**: Relay
+    - **relay_bracket.SLDPRT**: Bracket to hold the relay at a 90 degrees angle. This help with the electrical harness.
+    - **relay_assembly.SLDASM**: Assembly of the relay and its bracket
   - **roboclaw_2X30A.SLDASM**: Motor controller CAD. Got it on : [RoboClaw 2x30A Motor Controller (basicmicro.com)](https://www.basicmicro.com/RoboClaw-2x30A-Motor-Controller_p_9.html)
 
   
 
-- **Side pannel**
+- **Computer's side pannel**
 
   ![alt text](side_pannel.PNG)
 
-  - **computer.SLDPRT**: Box that represents the footprint of  Raspberry Pi 4 and a Jetson Nano ( sizes of their longest side with height of a Jetson)
-  - **side_pannel.SLDPRT**: Model of a panel that can be installed on the left-side slot of the pioneer to support a min-plate. In our case, we used is to support the computer.
+  - **computer.SLDPRT**: Box that represents the footprint of  Raspberry Pi 4 and a Jetson Nano ( sizes of their longest side with height of a Jetson). The CAD of the Raspberry Pi 4 is very heavy to load so this box is used as a lighter representation.
+
+  - **side_pannel.SLDPRT**: Model of a panel that can be installed on the left-side slot of the pioneer to support a small plate. In our case, we used is to support the computer.
+
   - **computer_support.SLDPRT**: Plate that can be screwed on the *side_pannel.SLDPRT* . It has the bolt pattern to install a Rasberry Pi 4 or a Jeston Nano.
 
+  - **computer_support_pi.SLDPRT**: Plate that can be screwed on the *side_pannel.SLDPRT* . It has the bolt pattern to install a Rasberry Pi 4. Was designed to have a more compact plate the selected computer is the Raspberry Pi 4. 
+
+  - **supported_pi.SLDASM**: Assembly of the side pannel and the Raspberry Pi 4 support plate.
+
+  - **fan_support.SLDPRT**: Support for the 5V fan that comes with the [LABIST Raspberry Pi 4 case]([LABISTS Case for Raspberry Pi 4](https://labists.com/products/raspberry-pi-4-case-kit)). Screwed with the screws that came with the kit.
+
+  - **fan_support_m3.SLDPRT**: Support for the 5V [CANAKIT Pi fan](https://www.canakit.com/raspberry-pi-fan.html). Screwed with M3 fasteners.
+
+    ![](C:\Users\etienne\OneDrive - USherbrooke\ecole\Uni\swarm\Pioneer-Mechanical-Update\fan_support.PNG)
+
+  **Under Voltage Lock Out's (UVLO) side pannel**
+
+  ![](C:\Users\etienne\OneDrive - USherbrooke\ecole\Uni\swarm\Pioneer-Mechanical-Update\uvlo.PNG)
+
+  - **UVLO_MODULE_PCB REV Project_PCB_Rev.SLDASM**: CAD of the Under Voltage Lock Out module made by the SwarmUS electrical team. Used to cut power whenever the battery is below a selected voltage threshold. [Link]([Electrical/04 - UVLO_MODULE at master · SwarmUS/Electrical (github.com)](https://github.com/SwarmUS/Electrical/tree/master/04 - UVLO_MODULE)) to project.
```

Comment:
Used to disconnect the battery whenever it is below the selected voltage threshold